documentation/source/user_section/tutorials/diffusion-weighted-mri/preprocessing.rst

```diff
@@ -54,12 +54,11 @@ Once the segmentation is obtained, we can use it to create a mask around the cor
 
 .. code::
 
-   sct_create_mask -i dmri_dwi_mean.nii.gz -p centerline,dmri_dwi_mean_seg.nii.gz -f cylinder -size 35mm
+   sct_create_mask -i dmri_dwi_mean.nii.gz -p centerline,dmri_dwi_mean_seg.nii.gz -size 35mm
```

Comment:
Why no `-f cylinder`? And do we need to change all other instances of `-f cylinder`? :thinking:

Reply:
I think now the default is cylinder